jupyterhub/spawner.py

```diff
@@ -174,12 +263,16 @@ def _deprecate_percent_u(self, proposal):
             v = v.replace('%U', '{username}')
             self.log.warning("Converting %r to %r", proposal['value'], v)
         return v
-    
-    disable_user_config = Bool(False,
-        help="""Disable per-user configuration of single-user servers.
-        
-        This prevents any config in users' $HOME directories
-        from having an effect on their server.
+
+    disable_user_config = Bool(
+        False,
+        help="""
+        Disable per-user configuration of single-user servers.
+
+        This prevents any config in users' $HOME directories from having an effect on their server.
```

Comment:
Perhaps, this would work :-)

```
Disable per-user configuration of single-user servers.

When starting the user's single-user server, any config file found in the user's $HOME directory will be ignored.
 
Note: a user could circumvent this if the user modifies their Python environment, such as when they have their own conda environments / virtualenvs / containers.
```